packages/components/src/query-controls/README.md

```diff
@@ -177,9 +177,10 @@ A function that receives the new author value. If not specified, the author cont
 -   Required: No
 -   Platform: Web
 
-#### `onCategoryChange`: `CategorySelectProps[ 'onChange' ] | FormTokenFieldProps[ 'onChange' ]`
+#### `onCategoryChange`: `( newCategory: string ) => void | FormTokenFieldProps[ 'onChange' ]`
 
 A function that receives the new category value. If not specified, the category controls are not rendered.
+The function's signature changes depending on whether multiple category selection is enabled or not.
```

Comment:
Just checking, does this line need to be added to the TS description as well?

Reply:
Good point, added in the latest commit